modules/sno-adding-worker-nodes-to-sno-clusters-manually.adoc

```diff
@@ -158,13 +158,17 @@ where:
 <hard_disk>:: Is the hard disk where you install {op-system}, for example, `/dev/sda`
 --
 
-.. For networks that have DHCP enabled, you do not need to set a static IP. Run the following `coreos-installer` command from the target host console to install the system:
+.. For networks that have DHCP enabled, you do not need to set a static IP. Instead, do the following: 
+
+... Run the following `coreos-installer` command from the target host console to install the system:
 +
 [source,terminal]
 ----
 $ coreos-installer install --ignition-url=<hosted_worker_ign_file> <hard_disk>
 ----
 
+... If the installation hangs instead of proceeding automatically, manually reboot the worker node.
```

Comment:
not sure i understand , coreos-installer command doesn't reboot the node as far as i remember so user will need to reboot it manually in anycase. Though command itself must finish.

Reply:
Thanks @tsorya . My understanding is that: 
- The command doesn't always finish, but sometimes nothing happens after it is entered. In which case the reboot is needed. 
- This happens in some platforms and not in others. (Less relevant.)

@anildhingra is my understanding correct? 

If so, I can change 'If the installation hangs' to 'If the command hangs" in the second bullet. If not, I can change it to "Wait for the command to complete, and then reboot the worker node". 
Thanks in advance.

Reply:
Didn't know we have such an issue. We run same logic in assisted-installer installation and never saw such issue. I just wonder how we can tell user that command actually wrote rhcos to the disk? I assume "hangs" means logging was stopped on 100% ?